frontend/app/views/spree/checkout/payment/_gateway.html.erb

```diff
@@ -1,27 +1,28 @@
 <%= image_tag 'credit_cards/credit_card.gif', id: 'credit-card-image' %>
 <% param_prefix = "payment_source[#{payment_method.id}]" %>
 
-<div class="field field-required">
+<div class="field field-required card_name" data-hook="card_name">
```

Comment:
I would prefer to not add additional `data-hook` attributes. They are a relict of the past.

Reply:
We can't remove the other `data-hook` from the same context (the other credit card fields). I think that keeping both here could be a good compromise to be minimum consistent. Not having the `data-hook` only in the card name will not enforce the deprecation in my opinion.

Reply:
I am not saying that an existing hook should be removed. As far as I can tell from the git diff a new `data-hook` is introduced.

Reply:
Yes you are right but my concern is that in the same file/form we would have:

```erb
<div class="field field-required card_name">
<div class="field field-required card_number" data-hook="card_number">
<div class="field field-required card_expiration" data-hook="card_expiration">
<div class="field field-required card_code" data-hook="card_code">
```

which is inconsistent. Anyway, I'm not too attached to this, I can live either way so if you think we should not add other `data-hook`s I'm ok asking @aitbw to remove it.

Reply:
I think @kennyadsl's reasoning makes sense for the sake of consistency and backwards compatibility (in case existing hooks are being used by stores)

Reply:
> sake of consistency and backwards compatibility 

Adding a new data-hook is not backwards compatibility, but anyway, this is not super important for me.